test/fixtures/eresolve-explanations.js

```diff
@@ -35,43 +35,45 @@ module.exports = {
       ],
     },
     peerConflict: {
-      name: '@isaacs/peer-dep-cycle-c',
-      version: '1.0.0',
-      whileInstalling: { name: '@isaacs/peer-dep-cycle-a', version: '1.0.0' },
-      location: 'node_modules/@isaacs/peer-dep-cycle-c',
-      dependents: [
-        {
-          type: 'peer',
-          name: '@isaacs/peer-dep-cycle-c',
-          spec: '1',
-          from: {
-            name: '@isaacs/peer-dep-cycle-b',
-            version: '1.0.0',
-            whileInstalling: { name: '@isaacs/peer-dep-cycle-a', version: '1.0.0' },
-            location: 'node_modules/@isaacs/peer-dep-cycle-b',
-            dependents: [
-              {
-                type: 'peer',
-                name: '@isaacs/peer-dep-cycle-b',
-                spec: '1',
-                from: {
-                  name: '@isaacs/peer-dep-cycle-a',
-                  version: '1.0.0',
-                  location: 'node_modules/@isaacs/peer-dep-cycle-a',
-                  dependents: [
-                    {
-                      type: 'prod',
-                      name: '@isaacs/peer-dep-cycle-a',
-                      spec: '1.x',
-                      from: { location: '/some/project' },
-                    },
-                  ],
+      peer: {
```

Comment:
Is this matching the new `peerConflict` object returning from arborist?  Is this all net new info cause it really seems like a breaking change from this perspective.